### PR TITLE
Filters are not applied to pagination

### PR DESCRIPTION
**Pull request by kaushikhande**
Fixes Issue: Filters are not applied to pagination #269 

Added filter params along with developer URL, so that filters are considered
for next page request.

### Pull request checklist

- [x] Your code contains tests relevant for code you modified
- [x] You have linted and tested the project with `bin/check`

### CODE REVIEW

File: app/queries/developer_query.rb [hunk 1]
```diff
@@ -12,6 +12,14 @@ def initialize(options = {})
     @role_types = options.delete(:role_types)
   end
 
+  def filters
+    @filters = {
+      sort: @sort || "",
+      time_zones: @time_zones || [],
+      role_types: @role_types || []
```
Comment (joemasilotti) [inline app/queries/developer_query.rb:19]: Why are we not using the `#sort`, `#time_zone`, and `#role_type` methods?

**Review by joemasilotti — CHANGES_REQUESTED** [review]
Thanks for the PR! Along with my comment, can you also add a test that ensures the "next" page links render correctly?

**Comment by kaushikhande** [comment]
> Thanks for the PR! Along with my comment, can you also add a test that ensures the "next" page links render correctly?

Okay

**Comment by joemasilotti** [comment]
@kaushikhande, thanks again for this PR. I had a business call out that pagination is broken when filtering. So I wanted to get this out ASAP and pushed a few commits.

**Comment by kaushikhande** [comment]
@joemasilotti Thank you for merging my PR. I can totally understand your urgency. I was on vacation for last 2 days. I look into it last night but was having some difficulty in completing the pagination tests. Thank you once again.

**Pull request merged**